examples/temp_linux.py

```diff
@@ -83,7 +83,8 @@ def echo_cmd(dev):
                            format(sub_cmd_in, sub_cmd_out))
 
 def die_temp(dev):
-    dev.cmd(MRPC_DIETEMP, struct.pack("<L", MRPC_DIETEMP_SET_MEAS))
+    # Comment out this next line for Gen4 switch
+    # dev.cmd(MRPC_DIETEMP, struct.pack("<L", MRPC_DIETEMP_SET_MEAS))
```

Comment:
First, this really doesn't belong in this PR which is supposed to be working on LTSSM stuff... This should be a separate PR.

Second, don't make broken code and don't leave code commented out. Probably best to find a way to check the generation and do the appropriate thing. Or maybe change the example to do a simpler MRPC command. (I only chose the temp one here because I thought it would be dead simple and stay that way).